src/azure-cli/azure/cli/command_modules/lab/_help.py

```diff
@@ -105,6 +105,8 @@
 """
 
 helps['lab environment delete'] = """
+[!CAUTION]
+Deleting resources in your lab is permanent, and cannot be undone.
```

Comment:
This statement almost applies to all Azure resources. I think this is already a common sense.

Reply:
@jiasli while I can understand that this may be common sense to some, we should not assume what another person does or does not already know.  This is part of an initiative with my supportability work to prevent more CritSits from being created for this exact problem.  So, with the data that I have for case creation this is not something that is inherently common sense to all.